staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -178,7 +178,7 @@ func newAdmissionMetrics() *AdmissionMetrics {
 				Subsystem:      subsystem,
 				Name:           "webhook_admission_duration_seconds",
 				Help:           "Admission webhook latency histogram in seconds, identified by name and broken out for each operation and API resource and type (validate or admit).",
-				Buckets:        []float64{0.005, 0.025, 0.1, 0.5, 1.0, 2.5},
+				Buckets:        []float64{0.005, 0.025, 0.1, 0.5, 1.0, 2.5, 10},
```

Comment:
according to the comment above this metric, admission webhooks timeout is set to 30 seconds by default, so I think it would make sense to also include 30, wdyt?

Reply:
I believe that comment is incorrect: `After the timeout passes, the webhook call will be ignored or the API call will fail based on the failure policy. The timeout value must be between 1 and 30 seconds. Default to 10 seconds.`

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts 

`The timeout for an admission webhook defaults to 10 seconds.`

Reply:
alright alright